demo/campaign/src/components/IndentedSectionGroup.tsx

```diff
@@ -7,18 +7,31 @@ export interface IndentedSectionGroupProps extends React.ComponentProps<typeof M
     cssClass?: string;
 }
 
-const CONTENT_WIDTH = theme.mailSize.contentWidth - theme.mailSize.contentSpacing * 2;
+export const indentedSectionGroupStyles = (
+    <MjmlStyle>{`
+      @media (max-width: 759px) {
+        .indented-section__group {
+          width: 100% !important;
+          max-width: 592px;
+        }
+      }
+    `}</MjmlStyle>
+);
+
+const CONTENT_WIDTH = theme.mailSize.contentWidth;
 
 export const IndentedSectionGroup = ({ children, cssClass, ...restProps }: IndentedSectionGroupProps) => {
     return (
         <MjmlSection
-            cssClass={`${cssClass || ""}`.trim()}
+            cssClass={(`${cssClass || ""}`.trim(), "indented-section")}
```

Comment:
I think this needs to be one string 🤔 

```suggestion
            cssClass={`indented-section ${cssClass || ""}`.trim()}
```